.github/workflows/Release.yaml

```diff
@@ -138,25 +148,19 @@ jobs:
 
 
   dispatch-documentation:
-    runs-on: ubuntu-22.04
     name: Dispatch documentation
     needs: [do-releases]
-    steps:
-      - name: Run documentation
-        # Using this actions cause others can have this problem:
-        # https://github.com/convictional/trigger-workflow-and-wait/issues/61
-        uses: mathze/workflow-dispatch-action@v1.2.0
-        id: run-documentation
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          ref: ${{ github.ref_name }}
-          fail-on-error: true
-          workflow-name: Documentation.yaml
-          payload: '{ "oci-image-name": "${{ inputs.oci-image-name }}"}'
-          use-marker-step: true
-          trigger-timeout: '30m'
-          run-id: dummy
+    uses: canonical/oci-factory/.github/workflows/Documentation.yaml@main
+    with:
+      oci-image-name: "${{ inputs.oci-image-name }}"
+    secrets: inherit
+
 
+  write-documentation-dispatch-summary:
+    runs-on: ubuntu-22.04
+    name: Write documentation dispatch summary
+    needs: [dispatch-documentation]
+    steps:
```

Comment:
same as above. no need for summary or URLs cause the job will run as part of the same run_id - ie. same workflow execution